playwright/mockData/users/RosaLuxemburg.ts

```diff
@@ -0,0 +1,10 @@
+import { ZetkinUser } from '../../../src/types/zetkin';
```

Comment:
Other mock data is in `index.ts` files in dedicated directories, while this is not. I prefer this, but maybe there is some reason to have the others in directories?

Reply:
Ah of course! That makes perfect sense.